react/Infos/index.jsx

```diff
@@ -1,67 +1,117 @@
 import React from 'react'
 import cx from 'classnames'
 import PropTypes from 'prop-types'
-import styles from './styles.styl'
-import Icon, { iconPropType } from '../Icon'
+import Icon from '../Icon'
 import Text, { SubTitle } from '../Text'
+import Stack from '../Stack'
+import palette from '../palette'
+
+import styles from './styles.styl'
 
-const Infos = ({ actionButton, icon, isImportant, text, className, title }) => {
+export const Infos = ({
+  description,
+  action,
+  dismissAction,
+  theme,
+  className
+}) => {
   return (
-    <div
-      className={cx(
-        styles['Infos'],
-        'u-stack-m',
-        isImportant ? 'u-bg-chablis' : 'u-bg-paleGrey',
-        className
+    <div className={cx(styles['Infos'], styles[`Infos--${theme}`], className)}>
+      <Stack spacing="m">
+        <div className={styles['Infos-description']}>
+          <Stack spacing="s">{description}</Stack>
+        </div>
+        {action && <div>{action}</div>}
+      </Stack>
+      {dismissAction && (
+        <button className={styles['Info-close']} onClick={dismissAction}>
+          <Icon icon="cross" color={palette['coolGrey']} />
```

Comment:
Is there a benefit for using `palette` over using `color="var(--coolGrey)"` ?

Reply:
Not that I'm aware, it seems to be the more widely used form though. Don't mind changing it if we'd rather move away from it.

Reply:
OK thanks. It was just for my culture ;)